### DISCUSSION TITLE
支持部分配置的默认设置

### DISCUSSION THREAD

**Discussion by cmpute** [hugo-next/hugo-theme-next#85] [Ideas]
我正准备从hexo迁移到hugo，但是配置迁移的挺麻烦的。我发现好像hugo的主题自己并不能很方便的有一套默认配置？不知能否在模板里面支持一些默认配置（比如`yearFormat`, `dateFormat`这种基本不会改的）？可以把默认配置放在`data/config.yaml`里面，然后在加载的时候如果用户没有提供设置则使用默认设置？Hugo的模板支持`$Scratch`变量，不知道会不会让这个更方便实现一点

**Comment by elkan1788** [comment]
Hugo默认是支持多环境的变量体系，你可以参考 [hugo-next-docs](https://github.com/hugo-next/hugo-next-docs) 的配置进行迁移，另外 Hexo 与 Hugo 本来就是不完全一样的，Hugo NexT 在设计时已经最大程度上考虑兼容性，如有更好的建议欢迎提出，也欢迎参与共建。
